### ISSUE TITLE
[a11y] accordion: lack of focus management using tab or shift+tab

### ISSUE THREAD

**Issue by Etat-Plateforme** [angular-ui/bootstrap#4250]
According to http://www.w3.org/TR/wai-aria-practices/#accordion when focus is on an accordion header, pressing the _tab_ key moves focus in the following manner:
1. If interactive glyphs or menus are present in the accordion header, focus moves to each in order.
2. When the corresponding panel is expanded (its aria-expanded state is 'true'), then focus moves to the first focusable element in the panel.
3. If the panel is collapsed (its aria-expanded state is 'false' or missing), OR, when the last interactive element of a panel is reached, the next Tab key press moves focus as follows:
* If a subsequent accordion panel is already expanded, focus moves to the first focusable element in this subsequent panel.
* If no subsequent accordion panel is expanded, focus moves to the first focusable element outside the accordion component.

**Comment by icfantv** [comment]
Closing in favor of #4772.
